app/waf/src/sequence.ts

```diff
@@ -83,10 +83,13 @@ export class MySequence implements SequenceHandler {
   }
 
   async authRequest(context: RequestContext) {
-    if (!process.env.PRODUCT_RELEASE) return;
-
     let {request} = context;
 
+    let ignoredAuthPaths = ['/openapi.json', '/explorer'];
+    for (let u of ignoredAuthPaths) {
+      if (request.path.startsWith(u)) return;
```

Comment:
If client request this path: /openapi.json', '/explorer', client will get nothing?

Reply:
In that case, the client can get a response without auth.

Reply:
Should we allow 'ping' without auth token? Or, which api endpoint should we use for healthcheck?

Reply:
The `ping` calls as3 API. I think that we should not allow anyone to `ping` without auth*. 
The client who wants to ping should have the auth info which is used for later works.

Reply:
A healthcheck call may not need to include openstack user token. That is not the topic of this pr, let us do that later.

Reply:
yea, we can add it when necessary.